### ISSUE TITLE
"Expected transform function" error when using with Reanimated

### ISSUE THREAD

**Issue by d4rky-pl** [software-mansion/react-native-svg#2054]
# Bug

When using react-native-svg with react-native-reanimated, animating SVG elements like `Circle` on the web results in a lot of console errors:

```
Error: <circle> attribute transform: Expected transform function, "undefined".
```

The animation itself works correctly.

## Environment info

Library version: 13.9.0

## Steps To Reproduce

1. Create `Svg` with `Circle` and animate any of its properties

Snack example: https://snack.expo.dev/@helpling/happy-apple

### Describe what you expected to happen:

The console.error should not be displayed

**Comment by GalaxyTea1** [comment]
yup, I also have the same problem 🐰

**Comment by jjspierx** [comment]
I was having this issue with 13.9.0.  Updated to 13.14.0 and errors are gone.

**Comment by oliviercperrier** [comment]
Same

**Comment by Karthik-B-06** [comment]
@jjspierx Yeah. I updated my package to the latest, which resolves this issue. 

But I get this warning, when using it with Expo. 

![image](https://github.com/software-mansion/react-native-svg/assets/35562287/4d4dfc9f-8b43-4a1c-b2e7-92530a255704)

**Comment by d4rky-pl** [comment]
@Karthik-B-06 this is fine as long as you're [using development build](https://docs.expo.dev/develop/development-builds/introduction/) (but will crash in Expo Go) and build your own dependencies. [You can opt-out of the warning for this library](https://docs.expo.dev/more/expo-cli/#configuring-dependency-validation).

**Comment by bohdanprog** [comment]
@d4rky-pl Hey,

Here is an [example](https://snack.expo.dev/@bohdan.artiukhov/animation-example) showing correctly how we should use Reanimated.
Additionally, I am sharing the Reanimated [documentation](https://docs.swmansion.com/react-native-reanimated/docs/core/useAnimatedProps/#adapters-) with you.

**Comment by d4rky-pl** [comment]
@bohdanprog I am not entirely sure how this is supposed to be helpful but... thank you?

**Comment by bohdanprog** [comment]
> @bohdanprog I am not entirely sure how this is supposed to be helpful but... thank you?

Case your example doesn't work?

**Comment by d4rky-pl** [comment]
@bohdanprog uh, yeah, that's usually what happens when you comment on an issue with a Snack from a year ago using an Expo version that is not supported by now 😅 

The example worked in May 2023 and I know how to use the library, the bug has already been fixed.

**Comment by bohdanprog** [comment]
I understand. I apologize for the lengthy response, but we are striving to do our best as we aim to improve. :)

Thank you for your response :)
